dev/wlp.lib.extract/src/wlp/lib/extract/SelfExtract.java

```diff
@@ -167,6 +175,15 @@ public static int doMain(String[] args) {
             } else {
                 outputDirFromUser = new File(targetString.trim());
             }
+
```

Comment:
Is the following better?
            
           String pathName = outputDirFromUser.getPath();
            if (pathName.trim().startWith("~")) {